envoy-control-runner/src/main/resources/application-docker.yaml

```diff
@@ -2,4 +2,8 @@ envoy-control:
   source:
     consul:
       host: consul
-      port: 8500
\ No newline at end of file
+      port: 8500
+
+chaos:
+  username: "user"
+  password: "{noop}pass"
```

Comment:
what is a `{noop}`?

Reply:
{noop} idicates password encryption - in this case plain text will be used.